Features/CopyMoveDelete/CopyContent.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Copy content using te role of Editor
+    In order to validate the copy action
+    As an Editor user
+    I need to be able to copy an object that I am viewing
+
+    Background:
+        Given I am logged in as an Editor in PlatformUI
+
+    ##
+    #Copy objects to valid locations
+    ##
+    @javascript
+    Scenario: Copy one object without children objects
+        Given a "Destiny" folder exists
+        And a "News Flash" article exists
+        When I copy the "News Flash" into "Destiny" folder
```

Comment:
We must think by _Context_ here, in my opinion.

Depending on the context, there are terms/concepts/steps that will be very accurate, and others that will be much more abstract, with very little details.

I'd say we need at least one test, somewhere, that details the action of copying a content from its full view. The scenario title should reflect this: "Copying a content using the button on the detail view". In this one, it should be very clear where we stand and what actions we take to perform the copy and ensure that the result is clear.

If another scenario, outside the "Copy" context, requires a copy, then it should be a oneliner, like "I copy Content X to Location Y (using the full view button)", since it is not what we are interested in. _How_ we copy (e.g. full view button) might matter, or might not, it depends on the context again: if _how_ we copy matters to the scenario, then it should be specified. If we are testing how feature X reacts to a copy of a content, then we could even do the copy using the Public API, we don't need to click on the buttons.

Does it make sense ?